app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class SessionsController < ApplicationController
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.find_by(name: params[:user][:name])
```

Comment:
User doesn't need to be an instance var here since you are redirecting in all cases.